### PR TITLE
Adds initial volunteers page.

### PR DESCRIPTION
**Pull request by ajessup**
In a recent SSC call, it was proposed we create a directory of volunteers for the SPIFFE project who may be willing to participate in certain duties on behalf of the project, such as booth duty or public speaking.

This PR is to create the initial directory, with the expectation that we would invite other community members who were willing to add their names via subsequent pull requests.

This initial PR also adds @dfeldman @amoore877 @ajessup and Sunil Ravipati who verbally acknowledged on the SSC call that they were willing to be included.

### CODE REVIEW

**Comment by ajessup** [comment]
@dfeldman @amoore877 if you'd like to update your details here just let me know

File: community/volunteers.md
Comment (amoore877) [inline community/volunteers.md]:
```suggestion
| Andrew Moore | [Slack Profile](https://spiffe.slack.com/team/UG75JQCR1) | Seattle, USA | Introduction, use cases, operational experience | Webinars, Speaking |
```

File: community/volunteers.md
Comment (amoore877) [inline community/volunteers.md]:
```suggestion
| Who | Contact | Location | Preferred speaking topics | Preferred duties | Volunteer Notes
```

additional column may be helpful here on other notes from the volunteer. Mine would be `Requires additional lead time for review and approval by employer`; basically I'm unlikely to be able to urgently fill in for something

File: community/volunteers.md
Comment (amoore877) [inline community/volunteers.md:11]:
```suggestion
| Andrew Moore | [Slack Profile](https://spiffe.slack.com/team/UG75JQCR1) | Seattle, USA | Introduction, use cases, operational experience | Webinars, Speaking | Requires additional lead time for review and approval by employer
```

part of comment got lost in updates

Comment (amoore877) [inline community/volunteers.md:11]:
```suggestion
| Andrew Moore | [Slack Profile](https://spiffe.slack.com/team/UG75JQCR1) | Seattle, USA | Introduction, use cases, operational experience | Webinars, Speaking | Requires additional lead time for review and approval by employer
```

part of comment got lost in updates

**Comment by amoore877** [comment]
+1 on PR (though I'm not a valid approver) and my entry (one comment)

thanks for putting this up @ajessup

**Review by amoore877 — APPROVED** [review]

**Comment by amoore877** [comment]
> +1 on PR (though I'm not a valid approver) and my entry (one comment)
> 
> thanks for putting this up @ajessup

github decided I can review now. cool

**Comment by elinesterov** [comment]
You can add me there :) if you need more folks. I won't be able to travel to KubeCon EU but I still have the deck we used on KubeCon US that might be helpful for folks who will be at the booth at Kubecon EU.

**Comment by ajessup** [comment]
Thanks @elinesterov! Mind opening up a quick PR to add your details to the list? No need to commit to a specific event to be included.

**Pull request merged**
